src/Microsoft.Sbom.Api/Executors/LicenseInformationService.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Serilog;
+
+namespace Microsoft.Sbom.Api.Executors;
+public class LicenseInformationService
+{
+    private readonly ILogger log;
+
+    public LicenseInformationService(ILogger log)
+    {
+        this.log = log ?? throw new ArgumentNullException(nameof(log));
+    }
+
+    public async Task<List<string>> FetchLicenseInformationFromAPI(List<string> listOfComponentsForApi)
+    {
+        int batchSize = 400;
```

Comment:
I _think_ the API allows up to 500. Is this just to be conservative with batches?

Reply:
I wasn't sure of the limit. Moved it up to 500 without an issue.